nirc_ehr/resources/queries/study/deaths.js

```diff
@@ -74,6 +80,13 @@ function onUpsert(helper, scriptErrors, row, oldRow) {
         //only allow death record to be created if animal is in demographics table
         if (idMap[row.Id]) {
 
+            var calc_status = undefined;
+            if (row.QCStateLabel.toUpperCase() === 'REQUEST: PENDING' || row.QCStateLabel.toUpperCase() === 'REVIEW REQUIRED')
+                calc_status = 'Necropsy Pending';
```

Comment:
Maybe I'm misunderstanding but I thought we decided we weren't going to do this. The animal would be left alive and reporting

Reply:
Missed to remove this, this was my initial implementation.